modules/mvt/docs/api-reference/mapbox-vector-tile-loader.md

```diff
@@ -34,12 +34,22 @@ const loaderOptions = {
 const geoJSONfeatures = await load(url, MVTLoader, loaderOptions);
 ```
 
+## Outputs
+
+### GeoJSON-like with local coordinates
```

Comment:
Yes, that is true. I didn't want to call it GeoJSON when returning local coordinates because the specification says WGS84 lng/lat coordinates.

That said, we can have a `geojson` flag that returns GeoJSON with local coords or WGS84 coords when true, and the vector-tile module internal format when false. But I think we can get rid of that flag and return GeoJSON always, I think that internal format is not that useful from my point of view and we can simplify things.

So we can have options for configuring that like:
```js
{
  coordinates: 'local' / 'wgs84',
  tileIndex: {
    x: 0,
    y: 0,
    z: 0
  }
}
```

That way, we would have only 2 configuration cases instead of 4. What do you think?